src/main/java/no/nav/folketrygdloven/beregningsgrunnlag/grenseverdi/FinnGrenseverdiUtenFordeling.java

```diff
@@ -61,12 +54,31 @@ public Evaluation evaluate(BeregningsgrunnlagPeriode grunnlag) {
 				resultater.put("inntektgraderingsprosent", grunnlag.getInntektsgraderingFraBruttoBeregningsgrunnlag());
 			}
 		}
+		
+		//hvis §8-47a, skaler med fast faktor
+		var erInaktivTypeA = MidlertidigInaktivType.A.equals(grunnlag.getBeregningsgrunnlag().getMidlertidigInaktivType());
+		if (erInaktivTypeA) {
+			BigDecimal reduksjonsfaktor = grunnlag.getBeregningsgrunnlag().getMidlertidigInaktivTypeAReduksjonsfaktor();
+			grenseverdi = grenseverdi.multiply(reduksjonsfaktor);
+			resultater.put("grad847a", reduksjonsfaktor);
+			grunnlag.setReduksjonsfaktorInaktivTypeA(reduksjonsfaktor);
+			
+			BigDecimal justertTotalUtbetalingsgradFraUttak = totalUtbetalingsgradFraUttak.multiply(reduksjonsfaktor);
```

Comment:
Kvifor ønsker i å justere utbetalingsgrad fra uttak her?

Reply:
Det er for å kunne vise i tilkjent yteles og sende til DVH hva utbetalingsgraden er (kombinert uttak + 8-47a-justeringen)